storage/src/lib.rs

```diff
@@ -71,71 +88,81 @@ pub fn aleo_ledger_dir(network: u16, dev: Option<u16>) -> PathBuf {
 }
 
 ///
-/// Returns the directory for accessing the operator files from Aleo storage.
+/// Returns the directory for accessing the prover files from Aleo storage.
 ///
-/// In production mode, the expected directory path is `~/.aleo/storage/operator-{network}`.
-/// In development mode, the expected directory path is `/path/to/repo/.operator-{network}-{id}`.
+/// In production mode, the expected directory path is `~/.aleo/storage/prover-{network}`.
+/// In development mode, the expected directory path is `/path/to/repo/.prover-{network}-{id}`.
 ///
-pub fn aleo_operator_dir(network: u16, dev: Option<u16>) -> PathBuf {
-    // Retrieve the starting directory.
-    let mut path = match dev.is_some() {
-        // In development mode, the operator is stored in the repository root directory.
-        true => match std::env::current_dir() {
-            Ok(current_dir) => current_dir,
-            _ => PathBuf::from(env!("CARGO_MANIFEST_DIR")),
-        },
-        // In production mode, the operator is stored in the `~/.aleo/` directory.
-        false => aleo_dir(),
-    };
+pub fn aleo_prover_dir(network: u16, dev: Option<u16>) -> PathBuf {
+    let mut path = base_storage_path(dev);
 
-    // Construct the path to the operator in storage.
+    // Construct the path to the prover in storage.
     match dev {
-        // In development mode, the operator files are stored in a hidden folder.
+        // In development mode, the prover files are stored in a hidden folder.
         Some(id) => {
-            path.push(format!(".operator-{}-{}", network, id));
+            path.push(format!(".prover-{}-{}", network, id));
             path
         }
-        // In production mode, the operator files are stored in a visible folder.
+        // In production mode, the prover files are stored in a visible folder.
         None => {
             path.push("storage");
-            path.push(format!("operator-{}", network));
+            path.push(format!("prover-{}", network));
             path
         }
     }
 }
 
 ///
-/// Returns the directory for accessing the prover files from Aleo storage.
+/// Returns the path for the primary-related BFT files.
 ///
-/// In production mode, the expected directory path is `~/.aleo/storage/prover-{network}`.
-/// In development mode, the expected directory path is `/path/to/repo/.prover-{network}-{id}`.
+/// In production mode, the expected directory path is `~/.aleo/storage/bft-{network}/primary`.
+/// In development mode, the expected directory path is `path/to/repo/.bft-storage-{network}/primary-{id}`.
```

Comment:
maybe we can move both the ledger and the BFT-related storage to a new `../repo/.storage` path :thinking:?